services/QuillLMS/spec/workers/admin_received_admin_upgrade_request_from_teacher_analytics_worker_spec.rb

```diff
@@ -10,12 +10,13 @@
     let!(:teacher) { create(:teacher) }
     let(:analyzer) { double(:analyzer) }
     let(:reason) { 'Please' }
+    let(:new_user) { [true, false].sample }
```

Comment:
This might be another good place to use a `[true, false].each do |new_user|` block.  Or, more precisely, in the `it` block on line 17.